functions.php

```diff
@@ -374,3 +374,138 @@ function wp_get_list_item_separator() {
     return __( ', ' );
   }
 }
+
+
+// Update Registration form and functions
+add_action('login_head', function () {
+  ?>
+      <style>
+          #registerform>p:first-child {
+              display: none;
+          }
+      </style>
+  
+      <script type="text/javascript" src="<?php echo site_url('/wp-includes/js/jquery/jquery.js'); ?>"></script>
+      <script type="text/javascript">
+          jQuery(document).ready(function($) {
+              $('#registerform > p:first-child').css('display', 'none');
+          });
+      </script>
+  <?php
+  });
+  
+  //Remove error for username, only show error for email only.
+  add_filter('registration_errors', function ($wp_error, $sanitized_user_login, $user_email) {
+      if (isset($wp_error->errors['empty_username'])) {
+          unset($wp_error->errors['empty_username']);
+      }
+  
+      if (isset($wp_error->errors['username_exists'])) {
+          unset($wp_error->errors['username_exists']);
+      }
+      return $wp_error;
+  }, 10, 3);
+  
+  
+  add_action('login_form_register', function () {
```

Comment:
I'd like a new workflow for adding a subscriber, not change existing workflow for all users.